test/addons/rook-cephfs/filesystem.yaml

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# yamllint disable-line rule:line-length
+# Source: https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/filesystem-test.yaml
```

Comment:
Any reason not to use the remote yaml and patch it like we do with other rook resources?

This make the changes explicit - with this we have to download and diff the yaml to find whats was changed, and there is no comment about changed settings.

Reply:
The YAML in rook also has a subvolumegroup resource, which I was trying to avoid. (single file in rook)

Reply:
Added the comment on modification...